gulpfile.js

```diff
@@ -326,10 +326,10 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + "/**/*.html"])
     .pipe(replace('[ios.latest-os-version]', '14.6'))
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
-    .pipe(replace('[ios.current-app-version]', '2.2.1'))
+    .pipe(replace('[ios.current-app-version]', '2.3.0'))
     .pipe(replace('[android.latest-os-version]', '11'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
-    .pipe(replace('[android.current-app-version]', '2.2.1'))
+    .pipe(replace('[android.current-app-version]', '2.3.0'))
```

Comment:
Latest Android 2.3 RC is [2.3.2-RC2](https://github.com/corona-warn-app/cwa-app-android/releases/tag/v2.3.2-RC2).